tests/test_misc.py

```diff
@@ -1,16 +1,19 @@
 # coding=utf-8
 from collections.abc import MutableMapping
 
+import pytest
+
 from lbz.misc import MultiDict, NestedDict, Singleton, error_catcher, get_logger
 
 
-def test_nested_dict():
+def test_nested_dict() -> None:
     nest = NestedDict()
     nest["a"]["b"]["c"]["d"]["e"] = "z"
     assert nest == {"a": {"b": {"c": {"d": {"e": "z"}}}}}
+    assert nest["a"]["b"]["c"]["d"]["e"] == "z"
```

Comment:
If the above is true, this line just tests that Python's `dict` is working correctly 😉 

I would say, it is not needed but it is just a thought.